data/links.json

```diff
@@ -125,6 +125,11 @@
         "href": "https://packet.csh.rit.edu",
         "icon": "material:check-box"
       },
+      {
+          "name": "HAROLD Audio Selector",
```

Comment:
This is the opposite end of the spectrum lol, I was thinking something more like `Audiophiler/Harold`, but this works for me.